types/instance.d.ts

```diff
@@ -353,4 +353,25 @@ export interface FastifyInstance<
    * Prints the representation of the internal radix tree used by the router
    */
   printRoutes(): string;
+
+  /**
+   *  Frozen read-only object registering the initial options passed down by the user to the fastify instance
+   */
+  initialConfig: Readonly<{
+    connectionTimeout?: number,
+    keepAliveTimeout?: number,
+    bodyLimit?: number,
+    caseSensitive?: boolean,
+    http2?: boolean,
+    https?: boolean | { allowHTTP1: boolean },
```

Comment:
`{ allowHTTP1 }` should also be wrapped in `Readonly` utility type, see https://github.com/fastify/fastify/blob/834a4e58de86d7996b50a4663fbf2b0fac4686ee/lib/initialConfigValidation.js#L29

Reply:
done!